### ISSUE TITLE
-disable-kvm is not a valid option

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#306]
Originally reported of Google Code with ID 250.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

gnt-cluster (ganeti v2.5.1) 2.5.1

<b>What distribution are you using?</b>

Debian 6 squeeze

<b>What steps will reproduce the problem?</b>
1. gnt-instance modify -H kvm_flag=disabled instance
2. gnt-instance reboot instance

<b>What is the expected output? What do you see instead?</b>

The expected output is a running instance with kvm disabled, i have a node without cpu virtualization.

The error:

Job 189408 for test.intra.actualys.fr has failed: Failure: command execution error:
Could not start instance for full reboot: Hypervisor error: Failed to start instance test.intra.actualys.fr: exited with exit code 1 (/usr/bin/kvm: invalid option -- '-disable-kvm'
)

<b>Please provide any additional information below.</b>

I can make it work by patching hv_kvm.py in hypervisor directory and change '-disable-kvm' by '-no-kvm' at line 557.


```
Originally added on 2012-07-11 09:29:28 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Thanks for the report, this seems like a change in the QEMU/KVM command line that we should handle correctly.
```
Originally added on 2012-07-19 15:46:36 +0000 UTC.

Added Labels: Type-Defect Hypervisor-KVM
Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
Indeed something has changed:

# kvm --version; kvm -h | grep -c disable-kvm
QEMU emulator version 1.0 (qemu-kvm-1.0 Debian 1.0+dfsg-8~bpo60+1), […]
0

```
Originally added on 2012-12-18 16:06:24 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-04 09:27:00 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
